compiler/tests-ocaml/lib-unix/common/dune

```diff
@@ -0,0 +1,23 @@
+(tests
+ (names
+  append
+  channel_of
+  cmdline_prog
+  reflector
+  rename
+  truncate
+  uexit
+  utimes)
+ (libraries ocaml_testing unix)
+ (flags
+  (:standard -no-strict-formats \ -strict-formats))
```

Comment:
Maybe we should change the default in an env stanza in compiler/tests-ocaml/dune

Reply:
done in #1824